src/server/cards/adventures/DistantLands.js

```diff
@@ -0,0 +1,9 @@
+import Card from 'cards/Card';
+
+export default class DistantLands extends Card {
+  static cost = new Card.Cost({ coin: 2 });
+  static types = new Set('Action', 'Reserve', 'Victory');
+  async onPlay(player) {
+    player.moveCard(this, player.playArea, player.mats.tavern);
+  }
```

Comment:
I don't see a getVpValue() function for this, it will be slightly tricky.
Up to now at the end of the game we move all cards into the player's deck (I don't see this being done with reserves yet) and then call getVpValue() on all of them.
Now we need to move it to the deck so other stuff that counts number of cards will work correctly (gardens, fairgrounds) so I suggest we simply set this.VP when we play it


``js
  async onPlay(player) {
    player.moveCard(this, player.playArea, player.mats.tavern);
   this.VP = 4;
}
```